docs/introduction.md

```diff
@@ -1,51 +1,77 @@
+detailed overview of the workflow is provided in the [methods](#methods).
 
 ## Background
 
-**FIXME** Background information on Peptidoglycan analysis.
+Peptidoglycan is a ubiquitous and essential component of the bacterial cell envelope and its synthesis is the target of
+beta-lactams, the most widely used antibiotics. Peptidoglycan fragments represent a key mediator during pathogenesis and
+acute and chronic inflammation and their perception by the host contributes to the regulation of neurodevelopmental and
+psychiatric disorders in animal models. Studying the structure and composition of peptidoglycan is therefore of
+paramount importance to understand antibiotic resistance, chronic inflammatory diseases and neurodevelopmental and
+psychiatric disorders.
+
+Traditionally, peptidoglycan analysis involved a manual inspection of individual mass spectra to identify peptidoglycan
+fragments based on their theoretical mass. PGFinder is an open-source Python package that can be run through Jupyter
+notebooks or at the command line to perform an unbiased and automated analysis of LC-MS data. This enables a consistent
+and reproducible workflow, opening new perspectives to develop "peptidoglycomics" as a new discipline.
 
 ## Methods
 
-```{mermaid}
-graph TD;
-
-    A[Monomer Masses Database] --> C[Match monomers]
-    B[Deconvoluted MS data] --> C([Match monomers -/+ ppm tolerance])
-    C --> D[Matched theoretical monomer masses]
-    D --> E([Calculate dimer & trimer masses])
-    D --> I([Calculate modified monomers, dimers & trimers])
-    D --> K([Match & annotate MS data -/+ ppm tolerance])
-    E --> F[Theoretical dimer & trimer masses library]
-    F --> G([Match dimers & trimers -/+ ppm tolerance])
-    G --> H[Matched theoretical dimers & trimers masses]
-    H --> I
-    I --> J[Theoertical modified monomers, dimers & trimers masses library]
-    J --> K
-    K --> L[Raw matched MS data]
-    L --> M([Consolidate in source decay products + salt adducts])
-    M --> N[Processed MS data]
-    N --> O([Write to CSV file])
-    style A fill:#914800,stroke:#000000
-    style B fill:#914800,stroke:#000000
-    style C fill:#914800,stroke:#000000
-    style D fill:#914800,stroke:#000000
-    style E fill:#009110,stroke:#000000
-    style F fill:#009110,stroke:#000000
-    style G fill:#009110,stroke:#000000
-    style H fill:#009110,stroke:#000000
-    style I fill:#002191,stroke:#000000
-    style J fill:#002191,stroke:#000000
-    style K fill:#910007,stroke:#000000
-    style L fill:#910007,stroke:#000000
-    style M fill:#910007,stroke:#000000
-    style N fill:#910007,stroke:#000000
-    style O fill:#910007,stroke:#000000
-```
+The identification of muropeptides was carried out using 4 successive steps, indicated by different colours (orange,
+green, blue, red, respectively). As a first step, observed masses in the dataset are compared to a list of theoretical
```

Comment:
This is not colourblind-friendly, but it's kind of OK. I suggest getting an editable version of the diagram in the repo somehow. Maybe svg. Maybe a future issue if the diagram would need to be re-done.